astronomy-pipeline/astronomy_transform_functions.py

```diff
@@ -0,0 +1,85 @@
+"""Functions for the astronomy API transform script."""
+
+import json
+import pandas as pd
+
+from astronomy_extract_functions import get_db_connection
```

Comment:
Nice re-using of the function

Reply:
Reduces boilerplate and follows DRY, LGTM.